things/service_test.go

```diff
@@ -24,11 +24,14 @@ const (
 	token      = "token"
 	token2     = "token2"
 	n          = uint64(10)
+	prefix     = "fe6b4e92-cc98-425e-b0aa-"
 )
 
 var (
-	thing   = things.Thing{Name: "test"}
-	channel = things.Channel{Name: "test"}
+	thingList  = [n]things.Thing{}
+	channel    = things.Channel{Name: "test"}
+	thsExtID   = []things.Thing{{ID: "fe6b4e92-cc98-425e-b0aa-000000000001", Name: "a"}, {ID: "fe6b4e92-cc98-425e-b0aa-000000000002", Name: "b"}}
```

Comment:
You could use the prefix here: `prefix + "000000000001"` and ``prefix + "000000000002"``

Reply:
done.. resubmitted the patch also.